src/main/resources/db/migration/V4__Create_products_table.sql

```diff
@@ -1,12 +1,14 @@
 CREATE TABLE IF NOT EXISTS products (
-                          id SERIAL PRIMARY KEY,
-                          name VARCHAR(255),
-                          thumbnail_image VARCHAR(255),
-                          image VARCHAR(255),
-                          available BOOLEAN DEFAULT FALSE,
-                          description VARCHAR(255),
-                          created_at TIMESTAMP NOT NULL DEFAULT now(),
-                          updated_at TIMESTAMP(6),
-                          seller_id BIGINT,
-                          FOREIGN KEY (seller_id) REFERENCES sellers(id)
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255),
+    thumbnail_image VARCHAR(255),
+    image VARCHAR(255),
+    available BOOLEAN DEFAULT FALSE,
+    description VARCHAR(255),
+    created_at TIMESTAMP NOT NULL DEFAULT now(),
+    updated_at TIMESTAMP(6),
+    category_id BIGINT,
+    FOREIGN KEY (category_id) REFERENCES categories(id),
+    seller_id BIGINT,
+    FOREIGN KEY (seller_id) REFERENCES sellers(id)
```

Comment:
when declaring constraints as foreign keys we usually move them all at the bottom of DDL, after all column names  were declared, 
but nowadays there is even better syntax that is supported by PSQL 16+:
```sql
CREATE TABLE IF NOT EXISTS products (
    ...
    category_id BIGINT REFERENCES categories(id),
    seller_id BIGINT REFERENCES sellers(id)
);
```
for more info check these links: 
- https://www.postgresql.org/docs/current/tutorial-fk.html#TUTORIAL-FK
- https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-FK

also good practice is to put columns in a logical order, most significant go first.
but, it's up to you do decide in which order to put them :)